specs/multiera/babbage/mod.cddl

```diff
@@ -115,4 +115,5 @@ babbage_format_aux_data = #6.259({
 babbage_auxiliary_data =
     shelley_format_aux_data     ; @name shelley
   / shelley_ma_format_aux_data  ; @name shelley_m_a
-  / babbage_format_aux_data     ; @name babbage 
\ No newline at end of file
+  / babbage_format_aux_data     ; @name babbage 
+babbage_mint = _CDDL_CODEGEN_EXTERN_TYPE_
```

Comment:
nit: maybe add a comment to specify that we did this to handle duplicate keys on preprod